react-native/training/src/app/screens/ListBooks/components/DetailBook/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Image, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import CustomText from '../../../../components/CustomText';
+import { defaultCoverPage } from '../../../../../constants/images';
+import { strings } from '../../../../i18n';
+
+import styles from './styles';
+
+function BookDetail({ navigation }) {
+  const book = navigation.getParam('book');
```

Comment:
We are sure that we want take from params the book in each render? I'd take form params and save it into the state component, to avoid call this in each render. This book never change